test/cmd/process.sh

```diff
@@ -88,6 +88,9 @@ os::cmd::expect_success_and_not_text "oc process -f '${required_params}' require
 os::cmd::expect_success_and_text "oc process -f '${required_params}' --value=required_param='`cat ${OS_ROOT}/test/testdata/multiline.txt`'" 'also,with=commas'
 os::cmd::expect_success_and_text "oc process -f '${required_params}' --param=required_param='`cat ${OS_ROOT}/test/testdata/multiline.txt`'" 'also,with=commas'
 
+# set template parameters from parameter file with multiline values
+param_file="${OS_ROOT}/test/testdata/template_required_params.env"
+os::cmd::expect_success_and_text "oc process -f '${required_params}' --param-file=${param_file} -o yaml" 'first$'
```

Comment:
i see it's covered in test/cmd/templates.sh

i wonder if we shouldn't combine test/cmd/templates.sh and test/cmd/process.sh, seems like we probably have overlap and it's not like process can do anything except work with templates.